commands/user

```diff
@@ -92,12 +92,13 @@ function cmd_show {
                                ${DBUS_ATTRIBUTES} "UserPrivilege" \
                                2>/dev/null \
           | awk '{gsub(/"/, "", $NF); print $NF}')
+
+    local role
     if [[ -z "${grp}" ]]; then
-      echo "Invalid user: ${user}" >&2
-      return 1
+      role="no-access"
+    else
+      role=$(group_to_role "${grp}")
```

Comment:
why don't you change the `group_to_role` subroutine instead?

Reply:
Because I don't want to add the possibility to set this stupid role. This commit makes it printable only when it set by some other ways.

Reply:
How modifying `group_to_role` could aid in 'setting this stupid role' ? You're clearly amending the `group_to_role`'s behavior but doing it externally. I don't like it.